script/unicode/Unicode/UnicodeSkipList.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+import Init.Data.Char.UnicodeSkipList
+import Unicode.Unicode
+import Unicode.Parse
+import Init.Data.Char.Basic
+
+open Std System IO FilePath FS
+
+namespace Char.UnicodeSkipList
+
+/-
+The following code creates the unicode skip list data structure for
+a given property. To understand it at a high-level, we provide an
+explanation in Init/Data/Char/UnicodeSkipList.lean
+
+Note that it is implicitly assuming that `ucd` is not any value
+of type `Array UnicodeData` but the result of parsing the UnicodeData.txt
+file. Likewise, we are implicitly assuming that `property` is not an
+arbitrary Boolean-valued function but rather one that checks for some
+specific attribute.
+
+In case where such implicit assumptions are violated, the verifier
+will reject the table was incorrectly created.
```

Comment:
```suggestion
In case where such implicit assumptions are violated, the verifier
will reject the table was incorrectly created.
```
Hard to understand here. Perhaps "will reject the table, reporting that it was incorrectly created"?